index.bs

```diff
@@ -2634,8 +2637,17 @@ Multiple [=constructor operations=] may appear on a given [=interface=].
 For each [=constructor operation=] on the [=interface=], there will be a way to attempt to
 construct an instance by passing the specified arguments.
 
-The prose definition of a [=constructor operation=] must either initialize the value passed as
-<b>[=this=]</b>, or throw an exception.
+The constructor description of a [=constructor operation=] must either initialize the value passed
```

Comment:
What does the "constructor description of a constructor operation" mean? Can we say "the constructor steps of each constructor operation"?

Also applies to "method description", etc.

Reply:
Hopefully clarified.